test/meson.build

```diff
@@ -0,0 +1,9 @@
+test_file = [
+    'testMain.cpp',
+    'testInitializer.cpp',
+]
+
+executable('tvgUnitTests',
+           test_file,
+           include_directories : headers,
+           link_with : thorvg_lib)
```

Comment:
If we use test() of meson script, we can run test with ninja command.
ex) `ninja -C build test` command.
```
unitTests = executable('tvgUnitTests',
           test_file,
           include_directories : headers,
           link_with : thorvg_lib)

test('Unit test', unitTests)

```